docs/examples.md

```diff
@@ -133,26 +134,34 @@ The regex used by default is:
 
 `[^a-zA-Z0-9_](?:t)(?:\\(|\\s)\\s*(?:(?:\'((?:(?:\\\\\')?[^\']+)+[^\\\\])\')|(?:"((?:(?:\\\\")?[^"]+)+[^\\\\])"))/g`
 
+### Command line
+```
+$ i18next /path/to/file/or/dir -p "(.*)"
+```
 
+### Gulp
+```js
+.pipe(i18next({ parser: '(.*)' }))
+```
 
-**Filter files and folders (cli)**
-
-<!-- TODO removed -->
-
-`i18next /path/to/file/or/dir --fileFilter '*.hbs,*.js' --directoryFilter '!.git'`
-
-In recursive mode, it will parse `*.hbs` and `*.js` files and skip `.git` folder. This options is passed to readdirp. To learn more, read [their documentation](https://github.com/thlorenz/readdirp#filters).
+## Filter files and folders
```

Comment:
This was also deprecated. See here: https://github.com/i18next/i18next-parser/blob/master/docs/migration.md#cli

Reply:
I removed this section.